src/components/map/DrawingTools.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
```

Comment:
This file is in `src/components/map/DrawingTools.tsx`, but what happens when there is another mapping component lets say NorthArrow. 

So the directory structure would be like:
```
map
├── DrawingTools.test.tsx
├── DrawingTools.tsx
├── NorthArrow.test.tsx
├── NorthArrow.tsx
└── styles.scss
```

So is that one styles.scss for every map component?

Reply:
We have options, if there is little styles among all the components, then one shared SCSS file would suffice, if each component had lots, then we can modularize them out so there would be `DrawingTools.scss` and `NorthArrow.scss`. Having 20 files all called `styles.scss` gets very confusing very quickly in my opinion.